### DISCUSSION TITLE
2023/12/26/2023-review/

### DISCUSSION THREAD

**Discussion by giscus** [MrWillCom/MrWillCom.github.io#10] [Announcements]
# 2023/12/26/2023-review/

https://blog.mrwillcom.com/2023/12/26/2023-review/

**Comment by xianyuYUAN** [comment]
will老师好厉害哈哈哈，因为前一阵搭自己博客的时候 用的就是你发布在hexo上的主题，所以也在一直关注你的博客呢，没想到最近竟然更新了欸，作品都好优雅，太喜欢了，看到这篇文章做的画廊页面，也好厉害，will老师加油，欢迎来我博客看看，我是咸鱼
xianyuuuan.space :)

**Reply by MrWillCom** [reply]
谢谢谢谢你喜欢我的主题！一起加油！！
